v1/topdown/strings.go

```diff
@@ -631,10 +643,10 @@ func builtinTrimSpace(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Ter
 		return iter(operands[0])
 	}
 
-	return iter(ast.StringTerm(trimmed))
+	return iter(ast.InternedStringTerm(trimmed))
 }
 
-func builtinSprintf(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+func builtinSprintf(b BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
```

Comment:
`b` is still unused, isn't it?

Reply:
Yeah, that must be a relic from development. I thought that would be caught by linting though 🤔 
Either way, I'm fixing.